src/GuPaymentTrait.php

```diff
@@ -567,8 +567,20 @@ public function updateCard($token)
      */
     public function refund($id)
     {
-        $iuguInovice = $this->findInvoice($id)->asIuguInvoice();
+        $iuguInvoice = $this->findInvoice($id)->asIuguInvoice();
 
-        return $iuguInovice->refund();
+        return $iuguInvoice->refund();
+    }
+
+    /**
+     * Duplicate an in progress invoice
+     * @param string $id the invoice id
+     * @param array $config configuration of the duplicate
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function duplicate($id,$config){
```

Comment:
Descer a chaves para seguir o padrão do pacote.
```suggestion
    public function duplicate($id,$config)
    {
```